src/base/components/Dashboard.js

```diff
@@ -0,0 +1,109 @@
+import React from 'react';
+import { connect } from "react-redux";
+
+class DashboardPage extends React.Component {
+
+  constructor(props) {
+    super(props);
+    this.state = {
+      sensorData: []
+    }
+  }
+
+  componentDidMount() {
+    setTimeout(() => {
+      this.setState({ sensorData: this.props.sensorData })
+    }, 3000);
+    document.getElementsByClassName('main')[0].style.width = "1300px";
```

Comment:
No styles in the code

Reply:
But the "main div" width is quite small and I have to increase it to adjust all table columns!

Reply:
You can add style into .css
But there no need. We will update the table like i said before. Currently we need only working code.

Reply:
my code editor unable to find any .css file ?
are css added through hosted links ?

Reply:
https://github.com/webRunes/Default-WRIO-Theme
But don't do any changes at the moment. Just remove styles in the code.

Reply:
Ok

Reply:
removed!